content/kb/troubleshooting-default-setting-not-persist-after-upgrade.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Troubleshooting: Longhorn default settings do not persist after upgrade"
+author: Chin-Ya Huang
+draft: false
+date: 2021-04-29
+categories:
+  - "setting"
+---
+
+## Applicable versions
+
+All Longhorn versions.
+
+## Symptoms
+
+I have modified Longhorn default settings, but they do not persist after upgrade.
```

Comment:
Add more details here. e.g., When upgrading Longhorn system via helm or Rancher App, the modified Longhorn default settings doesn't work... When modifying the default settings via `kubectl -n longhorn-system edit configmap longhorn-default-setting`, the modification won't be applied to Longhorn system.

Reply:
Thank you. Fixed.